lib/css/modules/_styleTweaks.scss

```diff
@@ -48,6 +48,13 @@ blockquote.twitter-tweet {
 	.edited-timestamp { font-weight: bolder; }
 }
 
+.res-colorblind {
+	#mail.havemail {
```

Comment:
The other is for NER mail, but just `.havemail` here might be enough specificity for both